main.go

```diff
@@ -385,6 +385,12 @@ func (a *autographer) initHSM(conf configuration) error {
 		return fmt.Errorf("error in initHSM from crypto11.Configure: %w", err)
 	}
 	if tmpCtx != nil {
+		var hsm signer.HSM
+		if os.Getenv("KMS_PKCS11_CONFIG") != "" {
+			hsm = signer.NewGCPHSM(tmpCtx)
```

Comment:
How does this interact with our integration testing? Could we get GCP versions of them?

Reply:
I'll look into this a bit. At the very least we should make sure we have the same coverage we currently have for AWS, but it looks to me like all of the integration tests run through [app-hsm](https://github.com/mozilla-services/autograph/blob/30d626b459b0bb407c71aa32decf2ebd15f6a055/docker-compose.yml#L36-L59), which [uses the softhsm](https://github.com/mozilla-services/autograph/blob/30d626b459b0bb407c71aa32decf2ebd15f6a055/tools/softhsm/autograph.softhsm.yaml#L10-L15). So after #993 we might have a little bit of coverage that runs through `AWSHSM`, but obviously it doesn't end up using the AWS pkcs11 library.